packages/toolpad-components/src/Paper.tsx

```diff
@@ -1,24 +1,43 @@
 import * as React from 'react';
-import { Paper as MuiPaper, PaperProps } from '@mui/material';
+import { Paper as MuiPaper, PaperProps as MuiPaperProps, Stack, StackProps } from '@mui/material';
 import { createComponent } from '@mui/toolpad-core';
 
-function Paper({ children, sx, ...props }: PaperProps) {
+export type PaperProps = MuiPaperProps & {
+  direction?: StackProps['direction'];
+  gap?: StackProps['gap'];
+};
+
+function Paper({ direction, gap, children, sx, ...props }: PaperProps) {
   return (
     <MuiPaper sx={{ padding: 1, ...sx }} {...props}>
-      {children}
+      <Stack direction={direction} gap={gap}>
+        {children}
+      </Stack>
     </MuiPaper>
   );
 }
 
 export default createComponent(Paper, {
+  layoutDirection: 'both',
   argTypes: {
     elevation: {
       typeDef: { type: 'number', minimum: 0 },
       defaultValue: 1,
     },
+    direction: {
```

Comment:
Shouldn't we allow users to build UI in a paper just like it is in a page? If you drag at the bottom it creates an extra row, if you drag to the side it creates an extra column?

I feel like these properties `direction` and `gap` are a step back.

Reply:
I agree, I'd expect the behaviour that Jan described

Reply:
I thought about that, I guess I had tried it before but gave up on it for some reason. 
Might not be super easy but I'm trying again, I think it should be possible.

Reply:
I've made it so we can build layouts in certain slots, in this case inside the Paper component.
For that I created a new slot type, `layout`.
Let me know what you think!